### PR TITLE
HTTPS Ports

### PR DESCRIPTION
**Pull request by andre2308**
On Installation you can change HTTPS Port. Look Issuse #116

---
This change is [Reviewable](https://reviewable.io/reviews/openhab/openhab-syno-spk/137)

Only the Last Commit is from Interresst

### CODE REVIEW

File: WIZARD_UIFILES/install_uifile_ger [hunk 1]
```diff
@@ -21,7 +21,7 @@
     "step_title": "Adminport",
     "items": [{
         "type": "textfield",
-        "desc": "W&auml;hle den HTTP Port f&uuml;r Openhab aus:",
+        "desc": "W&auml;hle den die Ports f&uuml;r Openhab aus:",
```
Comment (cniweb) [inline WIZARD_UIFILES/install_uifile_ger]: W&auml;hle die Ports

File: WIZARD_UIFILES/upgrade_uifile_ger [hunk 2]
```diff
@@ -21,7 +21,7 @@
     "step_title": "Adminport",
     "items": [{
         "type": "textfield",
-        "desc": "W&auml;hle einen HTTP Port f&uuml; openhab:",
+        "desc": "W&auml;hle den die Ports f&uuml;r Openhab aus:",
```
Comment (cniweb) [inline WIZARD_UIFILES/upgrade_uifile_ger]: W&auml;hle die Ports

**Comment by cniweb** [comment]
Thanks  @andre2308, closes #116

File: WIZARD_UIFILES/upgrade_uifile_ger (hunk 2 above)
Comment (pieterzen) [reply WIZARD_UIFILES/upgrade_uifile_ger]: I believe the attempt is to identify HTTP/HTTPS ports; Shouldn't it also not mention HTTP here? And shouldn't line 43 perhaps use this 'wahle die ports ..' text?

Comment (andre2308) [reply WIZARD_UIFILES/upgrade_uifile_ger]: Hey pieterzen, 
I have take a Picture from the Wizard. 

image

Comment (pieterzen) [reply WIZARD_UIFILES/upgrade_uifile_ger]: I see; So removing HTTP is sufficient. Thanks for feedback

**Pull request merged**